gems/pending/spec/util/mount/miq_generic_mount_session_spec.rb

```diff
@@ -2,25 +2,29 @@
 require 'util/mount/miq_generic_mount_session'
 
 describe MiqGenericMountSession do
-  context "#connect" do
-    before do
-      MiqGenericMountSession.stub(:raw_disconnect)
-      @s1 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
-      @s2 = MiqGenericMountSession.new({:uri => '/tmp/abc'})
+  let(:prefix) { File.join(Dir.tmpdir, "miq_") }
 
-      @s1.logger = Logger.new("/dev/null")
-      @s2.logger = Logger.new("/dev/null")
+  it "#connect returns a string pointing to the mount point" do
+    described_class.stub(:raw_disconnect)
+    s = described_class.new(:uri => '/tmp/abc')
+    s.logger = Logger.new("/dev/null")
+
+    expect(s.connect).to match(%r(\A#{prefix}\d{8}-\d{5}-\w+\z))
```

Comment:
Yes, when you pass in `nil` which we are in this case, it uses the default temp directory which is `Dir.tmpdir`.